git_machete/utils.py

```diff
@@ -295,3 +296,24 @@ def format_choice(c: str) -> str:
         else:
             return colored(c, AnsiEscapeCodes.ORANGE)
     return f" ({', '.join(map_truthy_only(format_choice, choices))}) "
+
+
+class EdgeColor(Enum):
+    DIM = auto()
+    UNDERLINE = auto()
```

Comment:
`UNDERLINE` is not an edge color :/// `UNDERLINE` is just a way to format text... which is a different level of abstraction than edge colors!